input/pagecontent/index.md

```diff
@@ -605,7 +612,7 @@ The following syntax can be used to assigning flags:
 
   `* identifier, identifier.system, identifier.value, name, name.family MS`
 
-#### Invariant Rules
+#### 🚧 Invariant Rules
```

Comment:
Here you give Invariant the 🚧, but above online 435 invariant gets the 🚫. I would say it qualifies as a 🚧 since we I don't think we are planning on changing the way we apply Invariant rules. The defining of Invariants could still change, as we were discussing yesterday.

Reply:
ok, changed